src/main/java/uk/gov/hmcts/ccd/v2/internal/controller/UIDefinitionController.java

```diff
@@ -95,4 +106,35 @@ public UIDefinitionController(@Qualifier(AuthorisedGetCriteriaOperation.QUALIFIE
 
         return ResponseEntity.ok(new UISearchInputsResource(searchInputs, caseTypeId));
     }
+
+    @GetMapping(
+        path = "/banners",
+        headers = {
+            V2.EXPERIMENTAL_HEADER
+        },
+        produces = {
+            V2.MediaType.UI_BANNERS
+        }
+    )
+    @ApiOperation(
+        value = "Get Banner information for the jurisdictions",
+        notes = V2.EXPERIMENTAL_WARNING
+    )
+    @ApiResponses({
+        @ApiResponse(
+            code = 200,
+            message = "Success",
+            response = UIBannerResource.class
+        ),
+        @ApiResponse(
+            code = 404,
+            message = "Banners not found"
+        )
```

Comment:
For better design, an empty list of banners should be returned with 200, rather than a 404.

Reply:
A 404 is meant to designate a case where a client refers to a resource by mistake, and it is client's fault to refer to that. It would be the case if the client was to ask for, say, banner object of b1, which wasn't present. However, when the user asks a list of resources, the list may be an empty list, and still the query is valid. A 4XX response would be wrong.

Reply:
Code is fine, it is returning empty banners result if banners not available for the passed jurisdiction references. only the documentation need to be removed 

@ApiResponse(
            code = 404,
            message = "Banners not found"
        )